src/templates/healthcert/parsers/vacHealthCertV1/parseInfo.ts

```diff
@@ -0,0 +1,28 @@
+import { Immunization, Location } from "../../types";
+import { SimpleImmunizationObject } from "../../memo/vacV1Memo";
+
+const SG_LOCALE = "en-sg";
+const dateFormatter = new Intl.DateTimeFormat(SG_LOCALE, {
+  /**
+   * Should not respect browser timezone but rather,
+   * force "UTC" timezone because dates in vacc certs do not have time
+   **/
+  timeZone: "UTC",
+  month: "long",
+  day: "numeric",
+  year: "numeric"
+});
+export const formatDate = (iso?: string): string => (iso ? dateFormatter.format(new Date(iso)) : "N/A");
+
+export const simplifyImmunizationObjectWithLocation: (
+  locations: Location[]
+) => (i: Immunization) => SimpleImmunizationObject = locations => {
+  return (immunization: Immunization): SimpleImmunizationObject => ({
+    vaccineName: immunization.vaccineCode.coding[0].display,
+    vaccineLot: immunization.lotNumber,
+    vaccinationDate: formatDate(immunization.occurrenceDateTime),
```

Comment:
Let's import the `isoToDateOnlyString()` function from here: https://github.com/Notarise-gov-sg/healthcert-renderer/blob/master/src/util/datetime.ts#L26-L38